python_modules/libraries/dagster-fivetran/dagster_fivetran_tests/experimental/test_resources.py

```diff
@@ -108,3 +108,72 @@ def test_basic_resource_request(
             poll_timeout=2,
             poll_interval=1,
         )
+
+
+@pytest.mark.parametrize(
+    "n_polls, succeed_at_end",
+    [(0, True), (0, False), (4, True), (4, False), (30, True)],
+)
```

Comment:
nit: add test ids

Reply:
Done in [e023cd6](https://github.com/dagster-io/dagster/pull/26061/commits/e023cd6b82353f3f51d4ebb884335b7d37252871)